frontend/packages/helm-plugin/integration-tests/support/pages/helm/helm-details-page.ts

```diff
@@ -36,4 +36,42 @@ export const helmDetailsPage = {
     cy.get('form strong').should('have.text', releaseName);
     cy.get(helmPO.uninstallHelmRelease.releaseName).type(releaseName);
   },
+  checkHelmTab: (name: string) => {
+    cy.get(`a[data-test-id="horizontal-link-${name}"]`).should('exist');
+  },
+  selectHelmTab: (name: string) => {
+    cy.get(`a[data-test-id="horizontal-link-${name}"]`)
+      .should('exist')
+      .click();
+  },
+  verifyHelmActionsDropdown: () => cy.byLegacyTestID('dropdown-button').should('be.visible'),
+  clickHelmActionButton: () => cy.byLegacyTestID('dropdown-button').click(),
+  verifyActionsInCreateMenu: () => {
+    cy.get('.pf-c-dropdown__menu-item')
+      .contains('Repository')
+      .should('exist');
+    cy.get('.pf-c-dropdown__menu-item')
+      .contains('Helm Release')
+      .should('exist');
+  },
+  clickCreateMenu: (createMenuOption: string) => {
+    cy.byLegacyTestID('dropdown-button')
+      .click()
+      .get('.pf-c-dropdown__menu')
+      .contains(createMenuOption)
+      .click();
+  },
+  clickHelmChartRepository: () => cy.byLegacyTestID('openshift-helm-charts').click(),
+  clickCreateHelmRelease: () => {
+    cy.byLegacyTestID('dropdown-button').click();
+    cy.get('.pf-c-dropdown__menu-item')
+      .contains('Helm Release')
+      .click();
```

Comment:
Same here, better to use `data-test-id` instead of class. Or check if you can make use of `selectByDropDownText` function.

Reply:
Updated the PR